README.md

```diff
@@ -20,44 +20,49 @@ which provides the following:
 
 ## Status
 
-Tendermint KMS is currently **ALPHA SOFTWARE AND UNAUDITED -- USE AT YOUR OWN RISK**.
+Tendermint KMS is currently *beta quality*. It has undergone one security audit
+with only one low-severity finding.
```

Comment:
It'd be nice to publish and link to this audit (or a subset thereof).

Reply:
Yes, I agree. Do you have a link to it?

Reply:
@jessysaurusrex is working on it, but I think it will take some time. we can add it later.